apps/neo_node/lib/neo_node/parser.ex

```diff
@@ -142,6 +144,77 @@ defmodule NeoNode.Parser do
     }
   end
 
+  def parse_application_log(%{"executions" => executions}) do
+    executions
+    |> Enum.map(&parse_execution/1)
+    |> List.flatten()
+    |> Enum.filter(&(not is_nil(&1)))
+  end
+
+  defp parse_execution(%{"notifications" => notifications}) do
+    Enum.map(notifications, &parse_notification/1)
+  end
+
+  defp parse_execution(_), do: nil
+
+  defp parse_notification(%{
+         "contract" => contract,
+         "state" => %{
+           "type" => "Array",
+           "value" => [
+             %{"type" => "ByteArray", "value" => "7472616e73666572"},
+             %{
+               "type" => "ByteArray",
+               "value" => address_from
+             },
+             %{
+               "type" => "ByteArray",
+               "value" => address_to
+             },
+             %{"type" => "ByteArray", "value" => value}
```

Comment:
The type here may be `Integer` for the `BigInteger` value instead of `ByteArray`, it is best probably to allow either type though.